backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/types/converter/impl/ElasticsearchFieldConverter.java

```diff
@@ -36,15 +36,28 @@ JsonElement convertFromDsl(Object value,
 	Object convertFromProjection(JsonElement value, FromIndexFieldValueConvertContext context);
 
 	/**
-	 * Determine whether another converter is DSL-compatible with this one.
+	 * Determine whether another converter's {@link #convertFromDsl(Object, ToIndexFieldValueConvertContext)}
+	 * method is compatible with this one's.
+	 * <p>
+	 * @see org.hibernate.search.engine.backend.document.spi.UserIndexFieldConverter#isConvertFromDslCompatibleWith(UserIndexFieldConverter)
 	 *
-	 * @see org.hibernate.search.engine.backend.document.spi.UserIndexFieldConverter#isDslCompatibleWith(UserIndexFieldConverter)
+	 * @param other Another {@link ElasticsearchFieldConverter}, never {@code null}.
+	 * @return {@code true} if the given converter is DSL-compatible.
+	 * {@code false} otherwise, or when in doubt.
+	 */
+	boolean isConvertFromDslCompatibleWith(ElasticsearchFieldConverter other);
+
+	/**
+	 * Determine whether another converter's {@link #convertFromProjection(JsonElement, FromIndexFieldValueConvertContext)}
+	 * method is compatible with this one's.
+	 * <p>
+	 * @see org.hibernate.search.engine.backend.document.spi.UserIndexFieldConverter#isConvertFromProjectionCompatibleWith(UserIndexFieldConverter)
 	 *
 	 * @param other Another {@link ElasticsearchFieldConverter}, never {@code null}.
 	 * @return {@code true} if the given converter is DSL-compatible.
```

Comment:
projection-compatible? In other javadocs, you even used the full method name here.

Reply:
Fixed by adding the full method name.